.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
           fetch-depth: 1
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: nightly-2020-05-05
+          toolchain: nightly
```

Comment:
I think it will be better if we add some dsscription why we changed this. Few months later someone may want to know the history.

Reply:
I searched other repos and it seems it's common to just use nightly.